src/transformState.js

```diff
@@ -61,8 +61,26 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
-function transformState(state, transforms) {
-  // write code here
+ function transformState(state, transforms) {
+  for (const key in transforms) {
```

Comment:
```suggestion
  for (const transform of transforms) {
```
Можно перебирать сразу свойства и тогда не надо будет использовать key.

Reply:
let's use for of loop